frontend/src/wordInteractions.js

```diff
@@ -53,21 +82,64 @@ function playerTypesWord () {
   typingForm.addEventListener('submit', (event) => {
     event.preventDefault();
     const typedSubmission = typingForm['player-input'].value;
-    WORDS_TYPED.push(typedSubmission);
-    if (WORDS_SEEN.includes(typedSubmission)) {
-      removeWordFromPage(typedSubmission);
+    const matchOnPage = document.getElementById(typedSubmission);
```

Comment:
this line can be deleted, it's never used

Reply:
Good catch!